cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -1146,6 +1146,28 @@ func (csr *ClusterStateRegistry) buildInstanceToErrorCodeMappings(instances []cl
 	return
 }
 
+// GetScaleUpRequests returns map of nodeGroupName -> ScaleUpRequest.
+func (csr *ClusterStateRegistry) GetScaleUpRequests() map[string]*ScaleUpRequest {
```

Comment:
This function has been introduced so that `ScaleDownCandidatesDelayProcessor` can get  `csr.scaleUpRequests` when evaluating scale down candidate nodes [here](https://github.com/kubernetes/autoscaler/pull/5729/files#diff-5d0598cafbe97c5bf254de21c9425430e21a392aadbedaccc5d78d0fcf169db4R59-R61).